### PR TITLE
Updated Strings to use public enums and broke out into files for read…

### PR DESCRIPTION
**Pull request by KaneCheshire**
…ability

### CODE REVIEW

File: iosHackthonApp/Modules/Shared/Strings/Sources/Strings/Strings.swift
Comment (Gmiller290488) [inline iosHackthonApp/Modules/Shared/Strings/Sources/Strings/Strings.swift:11]: Nice

**Review by Gmiller290488 — APPROVED** [review]

**Review by raynelle03 — APPROVED** [review]

**Comment by KaneCheshire** [comment]
I'll merge this after @raynelle03 merges her work otherwise it'll cause conflicts and I'm happy to fix them in mine

**Pull request merged**
